plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/preferences/RunOptionsDefaultsComponent.java

```diff
@@ -206,40 +214,31 @@ public void modifyText(ModifyEvent event) {
   }
 
   private void validate() {
+    // we set pageComplete to the value of `partial` if the fields are valid
```

Comment:
Is this true? Do we ever set page complete to true directly? That is, is partial is false when do we call setPageComplete(true)?

Reply:
We do [set it `true` directly](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/blob/4d930745128bb6f80024587e2c59aa69ce3e72ff/plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/preferences/RunOptionsDefaultsComponent.java#L303)